packages/components/src/components/Select/SelectItem.tsx

```diff
@@ -55,18 +55,27 @@ export type SelectItemGroupProps = {
      * The internal group ID of the select item group.
      */
     groupId: string;
+    /**
+     * The groups heading
+     */
+    heading?: string;
     /**
      * The data test id of the select item group.
      */
     'data-test-id'?: string;
 };
 
 export const SelectItemGroup = (
-    { children, groupId, 'data-test-id': dataTestId = 'fondue-select-item-group' }: SelectItemGroupProps,
+    { children, groupId, heading, 'data-test-id': dataTestId = 'fondue-select-item-group' }: SelectItemGroupProps,
     forwardedRef?: ForwardedRef<HTMLDivElement>,
 ) => {
     return (
         <div data-test-id={dataTestId} className={styles.group} ref={forwardedRef} key={groupId}>
+            {heading ? (
+                <div className={styles.groupHeading}>
+                    <span aria-label={heading}>{heading}</span>
```

Comment:
I think we don't need the aria label here

```suggestion
                    <span>{heading}</span>
```

Reply:
probably don't need the `span` even